CHANGES.md

```diff
@@ -0,0 +1,51 @@
+ringpop-node release notes
+==========================
+
+10.15.0
+-------
+
+**Feature: Parition Healing**
+* Partition healing implementation [#272](https://github.com/uber/ringpop-node/pull/272) [#273](https://github.com/uber/ringpop-node/pull/273)
+* Refactor in preperation for partition healing [#260](https://github.com/uber/ringpop-node/pull/260) [#259](https://github.com/uber/ringpop-node/pull/259) [#258](https://github.com/uber/ringpop-node/pull/258) [#266](https://github.com/uber/ringpop-node/pull/266)
+* Use a DiscoverProvider-function to discover the bootstrap hosts [#253](https://github.com/uber/ringpop-node/pull/253)
```

Comment:
We should either document it here what the feature does and why it's needed for the customer, or link it somewhere.

Basically, as a customer, I should know: "why would I care?" Release notes should give a link or a hint.